util/math.go

```diff
@@ -10,3 +17,19 @@ func Min(a ...int) int {
 	}
 	return min
 }
+
+// TryParseFloat takes an input string and validates that it is a valid finite
+// floating point number. The number is returned if valid, NaN if not
+func TryParseFloat(s string) (float64, error) {
```

Comment:
Then how about ParseNumericFloat?

Reply:
I feel like the current name is more accurate/descriptive. I did add another function below that takes in a float and does similar validation